src/plugins/orangehrmRecruitmentPlugin/Api/CandidateScheduleInterviewAPI.php

```diff
@@ -0,0 +1,314 @@
+<?php
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+
+namespace OrangeHRM\Recruitment\Api;
+
+use Exception;
+use OrangeHRM\Core\Api\V2\CrudEndpoint;
+use OrangeHRM\Core\Api\V2\Endpoint;
+use OrangeHRM\Core\Api\V2\EndpointResourceResult;
+use OrangeHRM\Core\Api\V2\EndpointResult;
+use OrangeHRM\Core\Api\V2\Exception\BadRequestException;
+use OrangeHRM\Core\Api\V2\Exception\ForbiddenException;
+use OrangeHRM\Core\Api\V2\Exception\RecordNotFoundException;
+use OrangeHRM\Core\Api\V2\RequestParams;
+use OrangeHRM\Core\Api\V2\Validator\ParamRule;
+use OrangeHRM\Core\Api\V2\Validator\ParamRuleCollection;
+use OrangeHRM\Core\Api\V2\Validator\Rule;
+use OrangeHRM\Core\Api\V2\Validator\Rules;
+use OrangeHRM\Core\Traits\Auth\AuthUserTrait;
+use OrangeHRM\Core\Traits\ORM\EntityManagerHelperTrait;
+use OrangeHRM\Core\Traits\Service\DateTimeHelperTrait;
+use OrangeHRM\Core\Traits\UserRoleManagerTrait;
+use OrangeHRM\Entity\Candidate;
+use OrangeHRM\Entity\CandidateHistory;
+use OrangeHRM\Entity\CandidateVacancy;
+use OrangeHRM\Entity\Interview;
+use OrangeHRM\Entity\WorkflowStateMachine;
+use OrangeHRM\ORM\Exception\TransactionException;
+use OrangeHRM\Recruitment\Api\Model\CandidateInterviewModel;
+use OrangeHRM\Recruitment\Service\CandidateService;
+use OrangeHRM\Recruitment\Traits\Service\CandidateServiceTrait;
+
+class CandidateScheduleInterviewAPI extends Endpoint implements CrudEndpoint
+{
+    use CandidateServiceTrait;
+    use UserRoleManagerTrait;
+    use AuthUserTrait;
+    use DateTimeHelperTrait;
+    use EntityManagerHelperTrait;
+
+    public const PARAMETER_CANDIDATE_ID = 'candidateId';
+    public const PARAMETER_INTERVIEW_NAME = 'interviewName';
+    public const PARAMETER_INTERVIEW_DATE = 'interviewDate';
+    public const PARAMETER_INTERVIEW_TIME = 'interviewTime';
+    public const PARAMETER_NOTE = 'note';
+    public const PARAMETER_INTERVIEWERS = 'interviewers';
```

Comment:
```suggestion
    public const PARAMETER_INTERVIEWERS = 'interviewerEmpNumbers';
```
shall we go like this? followed in performance module also

Reply:
done